frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.spec.tsx

```diff
@@ -55,13 +74,29 @@ describe('ToolbarOverflowMenu', () => {
   });
 
   it('does not render Report Issue button in overflow menu if it was disabled', () => {
-    vi.stubEnv('VITE_ENABLE_REPORT_ISSUE', 'false');
     render(<TestComponent defaultOpen />);
+
+    expect(screen.getByTestId('screensharing-button')).toBeVisible();
     expect(screen.getByTestId('layout-button')).toBeVisible();
     expect(screen.getByTestId('archiving-button')).toBeVisible();
     expect(screen.getByTestId('emoji-grid-button')).toBeVisible();
     expect(screen.queryByTestId('report-issue-button')).not.toBeInTheDocument();
     expect(screen.getByTestId('participant-list-button')).toBeVisible();
     expect(screen.getByTestId('chat-button')).toBeVisible();
   });
+
+  describe('ScreenSharingButton', () => {
+    it('is not rendered for mobile devices', () => {
+      (util.isMobile as Mock).mockImplementation(() => true);
+      render(<TestComponent defaultOpen />);
+
+      expect(screen.queryByTestId('screensharing-button')).not.toBeInTheDocument();
+    });
+
+    it('is rendered for desktop devices', () => {
+      render(<TestComponent defaultOpen />);
+
+      expect(screen.getByTestId('screensharing-button')).toBeVisible();
```

Comment:
Just curious, but why do we use different locators?  In the test above, we use `queryByTestId` instead...

Reply:
We use `queryByTestId` when we expect it to be a `null` or element that doesn't exist since it'll allow a `null` or an `element`. Whenever we use `getByTestId`, the testing library will throw an error.

Reply:
fyi this made me go crazy the first time when my tests were failing because of the reason Chris is describing 😅